styles.css

```diff
@@ -1,5 +1,6 @@
 @import '/Stylesheet/Components/avatar.css';
 @import '/Stylesheet/Components/alert.css';
+@import '/Stylesheet/Components/badge.css';
 
 @import '/Stylesheet/css/general.css';
 @import '/Stylesheet/css/reset.css';
```

Comment:
You should import your general and reset files at the top and components after them so that the styles in components could override the common styles if a conflict arises between them.

Reply:
Okay changing that, thank you